lib/handlers/api.ts

```diff
@@ -277,6 +285,29 @@ export class ApiHandler {
         }
     }
 
+    async handleLocalExecution(req: express.Request, res: express.Response, next: express.NextFunction) {
```

Comment:
Just to understand "local execution" is what in this instance? Is this local as in "on the local server itself" - the thing you said you'd comment out? I can see if being useful for testing, so another +1 on keep-it-behind-an-option.

Reply:
Execute on this instance yes, definitely just for testing. And I can see an option being useful, but I don't want users to accidentally turn it on, as nothing in this function is sanitized...

Reply:
Understood, but we have plenty of options that are "buyer beware" for debug etc. I think having it in an option makes sense, else how might we test it?

Reply:
added
```
# adds endpoint '/localexecution/:hash' for testing of remote execution
localexecutionEndpoint=false
```

to `etc/config/compiler-explorer.defaults.properties`
for this